learn/calling-java-code-from-ballerina.md

```diff
@@ -130,8 +130,8 @@ Compiling source
 ... 
 ...
 
-Creating balos
-	target/balo/sameera-yaml_package-any-0.1.0.balo
+Creating the BALA file
```

Comment:
This is a command output. Therefore it should be changed to `Creating bala`.

Reply:
Need to change the command output to this as saying "Creating  bala" is not meaningful. Will create an issue for that.

Reply:
@sameerajayasoma @hevayo